scripts/gen_i18n.py

```
@@ -638,29 +638,53 @@ def generate_strings_cpp(
+            offsets: List[int] = []
+            current_offset = 0
+            for s in lang_strings:
+                offsets.append(current_offset)
+                current_offset += len(s.encode("utf-8")) + 1
+            en_offsets = list(offsets)
+            blob_strings = lang_strings
+        else:
+            offsets = []
+            current_offset = 0
+            blob_strings = []
+            for i, (s, en_s) in enumerate(zip(lang_strings, en_strings)):
```

Ruff (0.15.12)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)